### ISSUE TITLE
Amass can not output to JSON anymore?

### ISSUE THREAD

**Issue by mfocuz** [owasp-amass/amass#967]
On latest release I see this error when trying to print to JSON
`flag provided but not defined: -json`

Looks like since version 4 this options is dissapeared, probably together with --nolocaldb

Is it intentional change?

**Comment by Mr-Lazzy** [comment]
Hi if you have updated your amass then it has been recently updated to v4 which doesn't support JSON file any more. Your can get the data using sqlite3 commands

**Comment by mfocuz** [comment]
a lot of ppl used json output to build this into pipelines, so now it becomes more "convenient" exporting data from sqlite to json. Please bring feature back, it was super useful.

**Comment by NeuroWinter** [comment]
I agree updating to v4 has broken almost all of my oneliners that I have used for a long time !
